exporter/README.md

```diff
@@ -9,6 +9,15 @@ The exporters below can be set up on any Linux-based system, but the instruction
 
 ## Installation
 
+### Upgrading
```

Comment:
@keithf4 I'd probably put upgrading in its own section outside of `Installation` but that's just my preference. Happy to go either way on that one